ecal-recon/src/main/java/org/hps/recon/ecal/cluster/ClusterPositionCorrection.java

```diff
@@ -14,29 +14,29 @@
 public final class ClusterPositionCorrection {
     //Parameterizations tested in MC using v3-fieldmap
     //Nov 2015
-      
+
     // Variables for electron position corrections.
```

Comment:
We may do this in the fullness of time, but I don't think this should hold up the current release.

Reply:
Code like this sets precedent for it to continue to be developed like this in the future. The organization of this code could be done in a way such that in the future all of these issues of which correction are applied could be simply resolved by our rather sophisticated interface to mysql. Seems like a waste to not leverage the power of that software for these corrections, as it would make future developments and new calibrations for new runs much more straight forward, that could potentially not even need any changes to the code, or at least very minimized changes.

Reply:
This is currently not holding up the release, as we still have many open PRs that are planned to go into the release are still progressing. We could take a little bit of time and make a more earnest push to refactor this code, as that is the description of the PR.